integration/samples/same-name/specs/es5-api.ts

```diff
@@ -1,12 +1,14 @@
 import { expect } from 'chai';
 import * as fs from 'fs';
 import * as path from 'path';
-import * as API from '../dist/esm5/testing.js';
-import * as APITesting from '../dist/esm5/testing/testing.js';
 
 describe(`@sample/same-name`, () => {
 
   describe(`esm5/testing.js`, () => {
+    let API;
+    before(() => {
+      API = require('../dist/esm5/testing.js');
+    })
```

Comment:
Prevents TypeScript compilation error for spec suite. Fails `before` hook in test run, if bundle files does not exist.